R/Rules-methods.R

```diff
@@ -2620,14 +2733,28 @@ setMethod("stopTrial",
 
       ## so can we stop?
       doStop <- ratio <= stopping@target_ratio
+
+      if (isTRUE(stopping@report_label == "default")) {
+        report_label <-
+          paste(
+            "TD",
+            x@targetRatio,
+            "for",
+            x@targetEndOfTrial,
+            "target prob"
+          )
+      } else {
+        report_label <- stopping@report_label
+      }
```

Comment:
same here: why do we have this `stopping@report_label == "default"`, I think we changed already to "". Also `report_label` should be places out of `if-else` clause. Also I do not see `x` object here, so you cannot use `x@<slotname>`.

Reply:
same here - not corrected yet.